spec/services/messaging/mobitel/sms_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe Messaging::Mobitel::Sms do
+  describe "#send_message" do
+    it "passes sanitised variable content to the API" do
```

Comment:
question: What is "sanitised variable content"? What does unsanitised content look like, and what's the expected behaviour?

Reply:
Whoops. This test is not necessary. I had removed it, must have added it back when trying to fix the bad merge.

We sanitise the SMS variables for BSNL. We don't do that for other vendors.